src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Penguin;
+import core.basesyntax.model.Shark;
+import core.basesyntax.model.Sparrow;
+import core.basesyntax.model.Tiger;
+import core.basesyntax.type.Animals;
+import core.basesyntax.type.Birds;
+import core.basesyntax.type.Fishes;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Animals tiggo = new Tiger("Tiggo", true);
+        tiggo.eat();
+        Zoo<Animals> newOne = new Zoo<>(tiggo);
```

Comment:
I think that we should have one zoo for animals, fishes and birds

Reply:
fixed